scss/_navbar.scss

```diff
@@ -186,13 +187,13 @@
     padding-top: subtract(var(--#{$prefix}navbar-nav-link-padding-y), var(--#{$prefix}navbar-border-width));
     margin-left: calc(-1 * var(--#{$prefix}navbar-nav-link-padding-x)); /* stylelint-disable-line function-disallowed-list */
     word-break: break-all;
-    outline-offset: $spacer;
-    @include transition($navbar-transition);
```

Comment:
TODO: double check that it must be indeed removed. More generally do we keep `@include transition`s or not.